examples/siro_sandbox/sandbox_app.py

```diff
@@ -698,7 +872,8 @@ def depth_to_rgb(obs):
             np.flipud(image) for image in debug_images
         ]
 
-        self._update_text()
+        if self._sandbox_state == SandboxState.SIMULATION:
+            self._update_text()
```

Comment:
I think we'll need help text during the tutorial in the future. Let's refactor this so that it's easy for us to add tutorial-specific help text in the future.